### PR TITLE
Small logging cleanup

### PR DESCRIPTION
**Pull request by quardbreak**
Here's branch with changes I done months ago. This was planned to be a configuration refactor, but I drop this idea and decided to cleanup some logging things:
* Logs always printed the station time because this proc use the `time_stamp()` function, which output game's time, instead server's. So I added another function specially for logs.
* Killed the investigation system because it was used almost nowhere. The panel itself now can be used to access main, href and qdel logs which stored in global variables.
* Seems I tried to fix variable protection. Not sure if this fixed now, but I tried to at least hide sensitive information, I hope.

### CODE REVIEW

File: code/_global_vars/configuration.dm [hunk 1]
```diff
@@ -1,9 +1,9 @@
 // Bomb cap!
 var/global/max_explosion_range = 14
 var/global/href_logfile        = null
-var/global/game_version        = "Nebula13"
+var/global/game_version        = "nebula-dev"
```
Comment (MistakeNot4892) [inline code/_global_vars/configuration.dm]: Why?

File: code/_global_vars/lists/objects.dm [hunk 2]
```diff
@@ -15,8 +15,19 @@ var/global/host = null //only here until check @ code\modules\ghosttrap\trap.dm:
...
+/// Vowels.
+var/global/list/vowels =                     list("a","e","i","o","u")
+/// Alphabet a-z.
+var/global/list/alphabet =                   list("a","b","c","d","e","f","g","h","i","j","k","l","m","n","o","p","q","r","s","t","u","v","w","x","y","z")
+/// Alphabet A-Z.
+var/global/list/alphabet_capital =           list("A","B","C","D","E","F","G","H","I","J","K","L","M","N","O","P","Q","R","S","T","U","V","W","X","Y","Z")
+/// Alphabet a-z without vowels.
+var/global/list/alphabet_vowelless =         list("b","c","d","f","g","h","j","k","l","m","n","p","q","r","s","t","v","w","x","z")
+/// Alphabet A-Z without vowels.
+var/global/list/alphabet_vowelless_capital = list("B","C","D","F","G","H","J","K","L","M","N","P","Q","R","S","T","V","W","X","Z")
```
Comment (MistakeNot4892) [inline code/_global_vars/lists/objects.dm]: What are these for? This list is redundant in view of `uppertext(alphabet_vowelless)` and numbers_alt isn't used at all that I can see.

File: code/_helpers/logging.dm [hunk 3]
```diff
@@ -2,11 +2,11 @@
 // will get logs that are one big line if the system is Linux and they are using notepad.  This solves it by adding CR to every line ending
 // in the logs.  ascii character 13 = CR
 
-var/global/log_end= world.system_type == UNIX ? ascii2text(13) : ""
+var/global/log_end = world.system_type == UNIX ? ascii2text(13) : ""
 
 
 /proc/error(msg)
-	to_world_log("## ERROR: [msg][log_end]")
+	to_world_log("## ERROR ##: [msg][log_end]")
```
Comment (afterthought2) [inline code/_helpers/logging.dm:9]: Please don't change these; they aren't here for aesthetics, they are here to aid automated searching and log processing.

File: code/_helpers/logging.dm [hunk 4]
```diff
@@ -22,14 +22,14 @@ var/global/log_end= world.system_type == UNIX ? ascii2text(13) : ""
...
+	to_world_log("## WARNING ##: [msg][log_end]")
 
 //print a testing-mode debug message to world.log
 /proc/testing(msg)
-	to_world_log("## TESTING: [msg][log_end]")
+	to_world_log("## TESTING ##: [msg][log_end]")
 
 /proc/game_log(category, text)
-	direct_output(diary, "\[[time_stamp()]] [game_id] [category]: [text][log_end]")
+	direct_output(global.world_main_log, "\[[time_stamp_system()]] [game_id] [category]: [text][log_end]")
```
Comment (afterthought2) [inline code/_helpers/logging.dm:32]: For admin stuff game time is more relevant. It's also best to not change this stuff unless necessary. We aren't the userbase for this system.

File: code/controllers/configuration.dm [hunk 5]
```diff
@@ -264,7 +265,10 @@ var/global/list/gamemode_cache = list()
 	)
 
 /datum/configuration/VV_hidden()
-	. = ..() | protected_vars
+	. = ..() + list("protected_vars", protected_vars)
```
Comment (afterthought2) [inline code/controllers/configuration.dm:268]: This is syntactically invalid.

File: code/controllers/configuration.dm [hunk 6]
```diff
@@ -264,7 +265,10 @@ var/global/list/gamemode_cache = list()
...
+	
+/datum/configuration/VV_static()
+	. = ..() + list("protected_vars", protected_vars)
```
Comment (afterthought2) [inline code/controllers/configuration.dm:271]: Also this.

File: code/datums/repositories/admin_pm.dm [hunk 7]
```diff
@@ -31,14 +31,15 @@ var/global/repository/admin_pm/admin_pm_repository = new()
...
+	var/time
 
-/datum/admin_privat_message/New(var/sender, var/receiver, var/message)
-	station_time = time_stamp()
-	src.message = message
-	src.sender = sender
-	src.receiver = receiver
+
+/datum/admin_privat_message/New(param_sender, param_receiver, param_message)
+	time = time_stamp_system()
```
Comment (afterthought2) [inline code/datums/repositories/admin_pm.dm:41]: This is definitely not correct. You shouldn't even be exposing server local time to users.

File: code/datums/repositories/attack_logs.dm [hunk 8]
```diff
@@ -21,7 +21,7 @@ var/global/repository/attack_logs/attack_log_repository = new()
 	var/message
 
 /datum/attack_log/New(var/mob/mob_attacker, var/mob/mob_victim, var/action_message)
-	station_time = time_stamp()
+	time = time_stamp_system()
```
Comment (afterthought2) [inline code/datums/repositories/attack_logs.dm:24]: Should be game time, same with most other in-game interactions. Won't comment on others.

File: code/modules/admin/admin.dm [hunk 9]
```diff
@@ -6,16 +6,18 @@ var/global/floorIsLava = 0
 ////////////////////////////////
 /proc/message_admins(var/msg)
 	msg = "<span class=\"log_message\"><span class=\"prefix\">ADMIN LOG:</span> <span class=\"message\">[msg]</span></span>"
-	log_adminwarn(msg)
```
Comment (afterthought2) [inline code/modules/admin/admin.dm:9]: Why? This is very much deliberate.

File: code/modules/admin/investigation.dm [hunk 10]
```diff
@@ -0,0 +1,34 @@
+#define INVESTIGATION_SUBJECT_MAIN "main"
+#define INVESTIGATION_SUBJECT_HREF "href"
+#define INVESTIGATION_SUBJECT_QDEL "qdel"
+
+/client/proc/investigation_panel(subject in list(INVESTIGATION_SUBJECT_MAIN, INVESTIGATION_SUBJECT_HREF, INVESTIGATION_SUBJECT_QDEL))
```
Comment (afterthought2) [inline code/modules/admin/investigation.dm:5]: This misses the point of this system, which is to filter the logs to something useful for actual admin work. Admins don't care about qdel logs, and it's not even clear they should access the full logs.

File: code/modules/admin/view_variables/view_variables_global.dm [hunk 11]
```diff
@@ -33,10 +32,13 @@
...
 /decl/global_vars/VV_locked()
-	return vars
+	. = ..() + vars
 
 /decl/global_vars/VV_hidden()
-	return protected_vars
+	. = ..() + list("protected_vars", protected_vars)
+
+/decl/global_vars/VV_static()
+	. = ..() + list("protected_vars", protected_vars)
```
Comment (afterthought2) [inline code/modules/admin/view_variables/view_variables_global.dm:41]: More syntactically invalid stuff.

File: code/modules/admin/view_variables/view_variables_global.dm [hunk 12]
```diff
@@ -33,10 +32,13 @@
 	return "(<a href='?_src_=vars;datumedit=\ref[src];varnameedit=[varname]'>E</a>) "
 
 /decl/global_vars/VV_locked()
-	return vars
+	. = ..() + vars
```
Comment (afterthought2) [inline code/modules/admin/view_variables/view_variables_global.dm:35]: This doesn't make sense.

File: code/modules/client/client_procs.dm [hunk 13]
```diff
@@ -341,8 +341,8 @@ var/global/list/localhost_addresses = list(
 
 	if ((player_age <= 0) && !(ckey in global.panic_bunker_bypass)) //first connection
 		if (config.panic_bunker && !holder && !deadmin_holder)
-			log_adminwarn("Failed Login: [key] - New account attempting to connect during panic bunker")
```
Comment (afterthought2) [inline code/modules/client/client_procs.dm:344]: Again, this is clearly deliberate.

**Comment by quardbreak** [comment]
Goes to Alaska.

**Pull request closed without merging**